src/items.rs

```diff
@@ -408,6 +408,122 @@ impl<'a> FmtVisitor<'a> {
         }
     }
 
+    pub fn visit_enum(&mut self,
+                      ident: ast::Ident,
+                      vis: ast::Visibility,
+                      enum_def: &ast::EnumDef,
+                      generics: &ast::Generics,
+                      span: Span)
+    {
+        // Empty enums are a bit weird (and not like empty structs), let's not touch them for now
+        if enum_def.variants.len() == 0 {
+            let result = self.snippet(span);
+            self.changes.push_str_span(span, &result);
```

Comment:
You don't need to do this, you can just return and the span will get automatically added the next time we try to format something.